johnsonandjohnson/Bodiless-JS
# fix(organisms): Fix Menu item overlap after mouseover

## Changes
- Fixed an issue when clicking a top menu item in preview/static lead to submenus stay open.
- Submenus are no longer automatically shown on `focus` instead 'space' or `enter` key has to be pressed to open a submenu.
- Better accessibility that follows best practices defined by w3.org [here](https://www.w3.org/WAI/tutorials/menus/flyout/).

## Test Instructions
- Confirm submenus don't stay open on top menu item click/focus in Preview/Static mode.
- Confirm submenus may be opened by pressing `space` or `enter` keys when a top menu item is focused.

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/762
